spec/tasks/db_schema_dump_spec.cr

```diff
@@ -0,0 +1,12 @@
+require "../spec_helper"
+
+describe Db::Schema::Dump do
+  it "generates a new sql dump file" do
+    Db::Schema::Dump.new.call
+
+    filename = Dir.entries(Dir.current).find { |e| e =~ /lucky_avram_dev_dump/ }.to_s
+    File.exists?(filename).should eq true
+    File.read(filename).should contain "PostgreSQL database dump"
+    File.delete(filename)
```

Comment:
This could cause some issues where if File.read(filename) fails the  file would not be deleted. What if instead you add a new arg to `Dump.new` that accepts a filename. It would default to `db/structure.sql` but for tests could be configured to `tmp/structure.sql`. Could also accept an ARGV.first for a custom location. Then there could be a `Spec.before_each` that deletes all the files in `tmp`. That way we can be sure it is always clean. This would also let us ignore `/tmp` in gitignore instead of ignoring a file pattern in the project directory

I think the initialize method would look something like this:

```crystal
def initialize(@dump_to : String)
end

def initialize
  @ dump_to = ARGV.first? || raise "Must pass a path to dump the db structure to"
end
```

Something like that I think. Thoughts?